src/app/app.component.ts

```diff
@@ -33,4 +38,39 @@ export class App {
   loadAppInfo() {
     this.store$.dispatch(new LoadAppInfo());
   }
+
+  configureAccessibility() {
+    if (this.platform.is('ios') && window && window.MobileAccessibility) {
+      window.MobileAccessibility.getTextZoom(this.getTextZoomCallback);
```

Comment:
Is this the correct usage of the plugin? Could we not use the ionic native wrapper?

```
import { MobileAccessibility } from '@ionic-native/mobile-accessibility';


constructor(private mobileAccessibility: MobileAccessibility) { }

...

if(this.mobileAccessibility. getTextZoom(this.getTextZoomCallback);
```

Reply:
Appending it to the global `window` object makes it invisible to Angular's DI system too

Reply:
That's what the docs say, but in reality it doesn't work.